adapters/sn2core/sn2core.go

```diff
@@ -300,7 +300,7 @@ func AdaptSierraClass(
 		Abi:     response.Abi,
 		AbiHash: crypto.StarknetKeccak([]byte(response.Abi)),
 
-		Compiled: coreCompiledClass,
+		Casm: coreCompiledClass,
```

Comment:
note: we might want to rename the variables related to the types as well, as it might be a bit confusing long term